src/Tizen.WebView/Tizen.WebView/BackForwardList.cs

```diff
@@ -0,0 +1,133 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.ComponentModel;
+using System.Collections.Generic;
+
+namespace Tizen.WebView
+{
+    /// <summary>
+    /// This class provides the properties of Back Forward list item of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardListItem
+    {
+        private IntPtr _item_handle;
+
+        internal BackForwardListItem(IntPtr handle)
+        {
+            _item_handle = handle;
+        }
+
+        public string GetURL()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_item_url_get(_item_handle);
+        }
+
+        public string GetTitle()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_item_title_get(_item_handle);
+        }
+
+        public string GetOriginalURL()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_item_original_url_get(_item_handle);
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Back Forward list of a specific WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class BackForwardList
+    {
+        private IntPtr _list_handle;
+
+        internal BackForwardList(IntPtr handle)
+        {
+            _list_handle = handle;
+        }
+
+        /// <summary>
+        /// Gets current item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetCurrentItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_current_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets previous item of the back forward list.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public BackForwardListItem GetPreviousItem()
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_back_forward_list_previous_item_get(_list_handle);
+            BackForwardListItem item = new BackForwardListItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Gets the back forward list count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public uint GetCount()
+        {
+            return Interop.ChromiumEwk.ewk_back_forward_list_count(_list_handle);
+        }
+        /// <summary>
+        /// Gets the list containing the items preceding the current item
+        /// limited by limit.
+        /// </summary>
+        /// <param limit="limit"> limit The number of items to retrieve.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public IList<BackForwardListItem> CopyBackItems(int limit)
```

Comment:
Why we need this API,
I thinks, it is just for C language.

Could you explain responsibility of this method?

Reply:
Copies the items preceding the current item limited by 'limit', and returns them as a list.
BackForwardList is dynamically changed by user action or javascript, this API returns a snapshot of the part of BackForwardList.